operations/app/src/modules/storage/main.tf

```diff
@@ -23,6 +23,14 @@ resource "azurerm_storage_account" "storage_account" {
   }
 }
 
+resource "azurerm_storage_account_customer_managed_key" "storage_key" {
+  count = var.environment == "test" ? 1 : 0 // Only use Vormetric keys in test
+  key_name = "${var.resource_prefix}-key"
```

Comment:
For continuity, do we want to put the number of bits (ie. 2048, 4096) into the key name?  Or is everything 4096 except for the one above?

Reply:
I'll ask if they can rename the key. Also going to ask about adding it to staging once we get the staging environment up.

Reply:
Going to work on getting the key changed in test, but the key names are clearer in staging, and will be in prod.